### PR TITLE
Fix CI Part 2

### PR DESCRIPTION
**Pull request by tweirtx**
Android deployment works in theory, @nholo1332 is gonna look at the iOS side soon.

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -1,9 +1,12 @@
 #!/usr/bin/env bash
+echo "Running on $TRAVIS_OS_NAME"
+
 if [[ "$TRAVIS_OS_NAME" == "osx" ]]
 then
   cd ios && fastlane release
 elif [[ "$TRAVIS_OS_NAME" == "linux" ]]
 then
+      export SUPPLY_VERSION_CODE=211
```
Comment (ofekashery) [inline deploy.sh]: Is it really necessary?

Comment (tweirtx) [reply deploy.sh]: Yes

Comment (tweirtx) [reply deploy.sh]: Yes

Comment (ofekashery) [reply deploy.sh]: 1. Why?
2. Cannot us get it from the gradle file?

Comment (tweirtx) [reply deploy.sh]: It wouldn't work for Fastlane. This fixes the issue.

**Pull request closed without merging**